src/structs/Item.js

```diff
@@ -578,6 +600,8 @@ export class Item extends AbstractStruct {
       this.deleted === right.deleted &&
       this.redone === null &&
       right.redone === null &&
+      this.linkedBy === null &&
```

Comment:
Types (ContentType) can't be merged. If the type is deleted, we can simply forget the reference.